app/views/stations/edit.erb

```diff
@@ -0,0 +1,14 @@
+<form action="/stations/<%=@station.id%>" method="post">
+  <h1>Edit Station Details </h1>
+  <input type="hidden" name="_method" value="PUT">
+  <h3>Current Name: <%=@station.name%></h3>
```

Comment:
Probably not necessary to provide the current name (and other attributes) in the labels. Like the way you're using it below as a value. That will pre-populate the form with the current value, and give the user the info they need. 👍